Pod/Classes/VOKManagedObjectMap.h

```diff
@@ -35,6 +37,16 @@
 #   define VOK_MAP_FOREIGN_TO_LOCAL(inputKeyPath, coreDataSelectorSymbol) [VOKManagedObjectMap mapWithForeignKeyPath:inputKeyPath coreDataKey:VOK_CDSELECTOR(coreDataSelectorSymbol)]
 #endif
 
+/**
+ *  Creates a map for a string to a number using a default NSNumberFormatter.
+ *
+ *  @param inputKeyPath             The foreign key to match with the local key.
+ *  @param coreDataSelectorSymbol   The local selector symbol
+ */
+#ifndef VOK_MAP_FOREIGN_STRING_TO_LOCAL_NUMBER
+#   define VOK_MAP_FOREIGN_STRING_TO_LOCAL_NUMBER(inputKeyPath, coreDataSelectorSymbol) [VOKManagedObjectMap mapWithForeignKeyPath:inputKeyPath coreDataKey:VOK_CDSELECTOR(coreDataSelectorSymbol) numberFormatter:[VOKManagedObjectMap vok_defaultNumberFormatter]]
+#endif
```

Comment:
My preference is still that, rather than adding this macro, we fold this behavior into the inherent mapper behavior.  That is, if the foreign value is a string and the local value is a number and there's no number formatter set, use the default number formatter.